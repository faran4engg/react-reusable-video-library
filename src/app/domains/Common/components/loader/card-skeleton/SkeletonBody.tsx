import { FC } from 'react';
import SkeletonLines from './SkeletonLines';

const SkeletonBody: FC = () => (
  <div className="flex space-x-4 animate-pulse">
    <div className="w-12 h-12 bg-gray-300 rounded-full dark:bg-gray-700"></div>
    <SkeletonLines />
  </div>
);

export default SkeletonBody;
