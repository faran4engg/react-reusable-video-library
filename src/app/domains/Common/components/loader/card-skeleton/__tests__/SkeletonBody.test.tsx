import { create } from 'react-test-renderer';

import SkeletonBody from '../SkeletonBody';

describe('SkeletonBody', () => {
  it('should render', () => {
    const component = create(<SkeletonBody />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
