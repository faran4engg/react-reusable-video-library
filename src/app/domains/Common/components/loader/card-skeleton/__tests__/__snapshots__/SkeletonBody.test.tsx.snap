// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonBody should render 1`] = `
<div
  className="flex space-x-4 animate-pulse"
>
  <div
    className="w-12 h-12 bg-gray-300 rounded-full dark:bg-gray-700"
  />
  <div
    className="flex-1 py-1 space-y-4"
  >
    <div
      className="h-4 bg-gray-300 dark:bg-gray-700 rounded w-3/4"
    />
    <div
      className="space-y-2"
    >
      <div
        className="h-4 bg-gray-300 rounded dark:bg-gray-700"
      />
      <div
        className="h-4 bg-gray-300 dark:bg-gray-700 rounded w-5/6"
      />
    </div>
  </div>
</div>
`;
